tests/components/brother/test_sensor.py

```diff
@@ -256,9 +256,7 @@
     await init_integration(hass)
 
     await async_setup_component(hass, "homeassistant", {})
-    with patch.object(
-        BrotherDataUpdateCoordinator, "_async_update_data", return_value=True
-    ) as mock_update:
+    with patch.object(Brother, "async_update", return_value=True) as mock_update:
```

Comment:
We should patch where the name is used not where it's defined. Patch `"homeassistant.components.brother.Brother.async_update`.